components/Animation.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled from 'styled-components/native';
+
+const Flower = styled.Image`
+  width: 200px;
+  height: 200px;
+  margin-top: 60px;
+`;
+
+const Animation = () => {
+  return <Flower source={require('../assets/animation_200_khg5z7e1.gif')} />;
```

Comment:
Super small thing, but next time you upload images to your project, try to change names to something shorter - we don't need this `_200_khg5z7e1` in the file name